src/src/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -136,28 +148,68 @@ public boolean canUseLocalCache() {
 
         return false;
     }
-
+        
     /**
-     * App needs to give permission to AccountManager to use broker.
+     * To verify if App gives permissions to AccountManager to use broker.
+     * 
+     * @throws DeveloperAuthenticationException
+     * @return boolean If the GET_ACCOUNTS, MANAGE_ACCOUNTS, USE_CREDENTIALS permissions are granted in the Manifest.xml
      */
     private boolean verifyManifestPermissions() {
-        PackageManager pm = mContext.getPackageManager();
-        boolean permission = PackageManager.PERMISSION_GRANTED == pm.checkPermission(
-                "android.permission.GET_ACCOUNTS", mContext.getPackageName())
-                && PackageManager.PERMISSION_GRANTED == pm.checkPermission(
-                        "android.permission.MANAGE_ACCOUNTS", mContext.getPackageName())
-                && PackageManager.PERMISSION_GRANTED == pm.checkPermission(
-                        "android.permission.USE_CREDENTIALS", mContext.getPackageName());
-        if (!permission) {
+        final String methodName = ":verifyManifestPermissions";
+        final PackageManager packageManager = mContext.getPackageManager();
+        List<String> permerssionMissing = new ArrayList<String>();
+        Logger.v(
+                TAG + methodName,
+                "Broker related permissions checking starts.");       
+               
+        if(PackageManager.PERMISSION_GRANTED != packageManager.checkPermission(
+                "android.permission.GET_ACCOUNTS", mContext.getPackageName()))
+        {
```

Comment:
As we talked in today's scrum, we're going to keep the coding style with current file. Then we need to keep the curly brace in the same line. Also, there should be a space after if ()
You can check the java coding convention.